pkg/authorization/authorizer/attributes_builder.go

```diff
@@ -21,7 +21,7 @@ func (a *openshiftAuthorizationAttributeBuilder) GetAttributes(req *http.Request
 	// any url that starts with an API prefix and is more than one step long is considered to be a resource URL.
 	// That means that /api is non-resource, /api/v1 is resource, /healthz is non-resource, and /swagger/anything is non-resource
 	urlSegments := splitPath(req.URL.Path)
-	isResourceURL := (len(urlSegments) > 1) && a.infoResolver.APIPrefixes.Has(urlSegments[0])
+	isResourceURL := (len(urlSegments) > 2) && a.infoResolver.APIPrefixes.Has(urlSegments[0])
```

Comment:
This change was necessary because the previous rules were incorrectly identifying endpoints like `/api/v1` as a resource URL. No valid resource URL can be just two segments long, so changing the minimum length to 2 should be valid.